README.md

```diff
@@ -35,131 +35,54 @@
-
-### **Vulnerability scanners**
-
-- [beefproject/beef](https://github.com/beefproject/beef) is short for The Browser Exploitation Framework. It is a penetration testing tool that focuses on the web browser.
-- [ajinabraham/OWASP-Xenotix-XSS-Exploit-Framework](https://github.com/ajinabraham/OWASP-Xenotix-XSS-Exploit-Framework) is an advanced Cross Site Scripting (XSS) vulnerability detection and exploitation framework.
-- [Netflix-Skunkworks/sleepy-puppy](https://github.com/Netflix-Skunkworks/sleepy-puppy) is a cross-site scripting (XSS) payload management framework which simplifies the ability to capture, manage, and track XSS propagation over long periods of time.
-- [echo-devim/xbackdoor](https://github.com/echo-devim/xbackdoor) is a tool to take advantage of a persistent XSS vulnerability.
-
-### **Browser spoofing**
-
-Browser spoofing - is a technique that helps users fake that they are using a different browser configuration by changing the browsers features.
-
-The BotD helps to detect the following types of spoofing:
-- User Agent spoofing
-- Operating System spoofing
-- Hardware spoofing
-- etc.
-
-### **VM detection**
-
-The BotD helps to detect if the browser is running inside one of the popular virtual machines, like VirtualBox, VmWare, Parallels, Hyper-V, etc.
-
-### **Search bots**
-Google Bot, Bing Bot, Baidu Spider, Yahoo Bot, Alexa Bot, Apple Bot, Facebook Bot, Twitter Bot, Pinterest Bot,
-DuckDuckGo Bot, Coccoc Bot, Yandex Bot, Telegram Bot, Kiwi Status Spider, Naver Spider, Sputnik Bot, Petal Bot,
-Aspiegel Bot, Seznam Bot, Sogou Bot, DuckDuckGo Bot, Rackspace Bot, Pingdom Bot, WebPageTest.org crawlers,
-StatusCakeBot, Nutch-based Bot, Genieo Web filter, etc.
-
-<small><i>Many more tools and configurations are supported</i></small>
+-   Headless Browsers ([Chrome](https://www.google.com/chrome/), [Firefox](https://www.mozilla.org/en-US/firefox/new/))
+-   [SeleniumHQ/selenium](https://github.com/SeleniumHQ/selenium) is an umbrella project encapsulating a variety of tools and libraries enabling web browser automation.
+-   [microsoft/playwright](https://github.com/microsoft/playwright) is a Node.js library to automate Chromium, Firefox and WebKit with a single API.
+-   [ariya/phantomjs](https://github.com/ariya/phantomjs) is a headless WebKit scriptable with JavaScript.
+-   [segmentio/nightmare](https://github.com/segmentio/nightmare) is a high-level browser automation library.
+-   [electron/electron](https://github.com/electron/electron) framework lets you write cross-platform desktop applications using JavaScript, HTML and CSS.
+-   [laurentj/slimerjs](https://github.com/laurentj/slimerjs) is a scriptable browser.
 
 ### Documentation links:
-- #### [Browser API](docs/api.md)
-- #### [Server API](docs/server_api.md)
-- #### [Error Handling](docs/error.md)
-- #### [FAQ](https://github.com/fingerprintjs/botd/wiki/FAQ)
+
+-   #### [Browser API](docs/api.md)
```

Comment:
We don't have browser API for fully open-source BotD

Reply:
Probably you confuse something. This library will have no server API because it will be fully client-side, but it can't have no browser API.

Reply:
yeah, I thought it's about server API, sorry